Automattic/wp-calypso
# A4A: Sites Dashboard - Use php_version instead of php_version_num in our Hosting feature

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/284

## Proposed Changes

* This PR changes the field used by our Hosting feature from `php_version_num` to `php_version`. Given that PHP follows semantic versioning, is more appropriate to use a string type for this purpose. 

## Testing Instructions

* Apply diff D145191-code and point your hosts file to your sandbox
* Run `yarn start-a8c-for-agencies`
* Go to `http://agencies.localhost:3000/sites`
* Click over a site to open the preview pane
* The "Hosting" tab should display the correct PHP version for your site.
* Verify that the PHP version information shown in the hosting tab matches that from the endpoint. You can check this by inspecting the `/sites` endpoint in your browser's inspector, under the Network tab.

## Pre-merge Checklist
